### ISSUE TITLE
Ubuntu 20.04 QT scaling issue

### ISSUE THREAD

**Issue by danweecc** [ExpertSDR3/ExpertSDR3-BUG-TRACKER#230]
EESDR3 0.11.0 alpha 30122021
Ubuntu 20.04
Native screen resolution 1920 x 1080, No fractional scaling

EESDR3 starts with QT scaled up as if it was a Hi-DPI display and everything looks too large:-
![image](https://user-images.githubusercontent.com/13930571/149779989-4fcb947b-7abb-40ac-bfe6-df4cb1f7a15b.png)
![image](https://user-images.githubusercontent.com/13930571/149780029-eb2e8cf3-95a6-434f-9ccb-58b1431ef2b2.png)

If I set QT_SCALE_FACTOR=0.5, I get a slightly better initial screenL-
![image](https://user-images.githubusercontent.com/13930571/149779903-3abed6f6-da68-47ef-a02f-ef4d8b248ec5.png)

But after that, everything is still too large. I don't know if there needs to be QT_SCALE_FACTOR and QT_AUTO_SCREEN_SCALE_FACTOR=0 that needs to be set somewhere. As it is, it takes up too much screen space and is unusable. This is almost certainly related to the QT scaling but there is nothing the user can do to bypass this so it probably needs to be set within EESDR3 itself.

**Comment by ExpertSDR3** [comment]
dublicate #144

**Comment by LA5OUA** [comment]
Hi
I'm experiencing the same issue.
I'm also using Ubuntu 20.04.
ESDR3 0.12.1
Screen resolution 1920x1080
I have also played with QT_SCALE_FACTOR and similar parameters, but it only affect the initial screen. If I open the radio or firmware settings button everything is extremly large like the pictures danweecc shows.
I know this is related to qt itself, but I'm not able to find a workaround for this.
I have tried to use the built-in "application scale" in ESDR3, but it does not allow me to scale any smaller than 70% (and that is still about twice the size of the screen).
I guess more people are going to run in to the same problem in the future, so I suggest that you modify the scaling options to allow scaling down to 20% (so the users don't have to use a lot of time to find out how to manipulate qt).
